lib/interfaces/http-module.interface.ts

```diff
@@ -14,6 +15,6 @@ export interface HttpModuleAsyncOptions
   useFactory?: (
     ...args: any[]
   ) => Promise<HttpModuleOptions> | HttpModuleOptions;
-  inject?: any[];
+  inject?: InjectionToken[];
```

Comment:
I believe that we should reuse the `InjectionToken` from `@nestjs/common` instead of duplicating it here